archive/unarchive.go

```diff
@@ -33,3 +36,28 @@ func UnarchiveToDirectory(archive, destDir string) error {
 
 	return nil
 }
+
+func ExtractFileToDirectory(archive, destDir, fileName string) error {
```

Comment:
Just curious was it possible to extract both .tar and tar.gz before?

Reply:
Yes, and it still is. `archiver` looks at file extension and selects the appropriate extractor to use (seehttps://github.com/mholt/archiver/blob/v3.5.1/archiver.go#L458).